frontend/www/js/omegaup/components/login/Signup.vue

```diff
@@ -6,7 +6,7 @@
     <div class="card-body">
       <form>
         <div class="row justify-content-md-center">
-          <div class="col-md-4 col-md-offset-2">
+          <div class="col-md-4 col-md-offset-2 introjsusername">
```

Comment:
Para que tenga consistencia con el resto de las clases, ¿Se puede utilizar la convención [`kebab-case`](https://www.pluralsight.com/blog/software-development/programming-naming-conventions-explained#kebab-case) ? es decir, separar las palabras con un guión medio:
```suggestion
          <div class="col-md-4 col-md-offset-2 introjs-username">
```
`introjs` lo podemos dejar junto porque es el nombre de la librería.

Lo mismo para todas las demás clases que agregaste